examples/streaming.py

```diff
@@ -3,33 +3,42 @@
 from tweepy.streaming import StreamListener
 from tweepy import OAuthHandler
 from tweepy import Stream
+from tweepy import API
+
+
+# Override tweepy.StreamListener to add logic
+# both default methods on_status and on_error 
+# are overridden with added logic
+class MyStreamListener(StreamListener):
+    # A listener handles tweets that are received from the stream.
+    # This is a basic listener that just prints received tweets to stdout.
+    def on_status(self, status):
+        print(status.text)
+
+    def on_error(self, status):
+        print(status)
+
 
 # Go to http://apps.twitter.com and create an app.
 # The consumer key and secret will be generated for you after
-consumer_key=""
-consumer_secret=""
+consumer_key = ""
+consumer_secret = ""
 
 # After the step above, you will be redirected to your app's page.
 # Create an access token under the the "Your access token" section
-access_token=""
-access_token_secret=""
-
-class StdOutListener(StreamListener):
-    """ A listener handles tweets that are received from the stream.
-    This is a basic listener that just prints received tweets to stdout.
-
-    """
-    def on_data(self, data):
-        print(data)
-        return True
+access_token = ""
+access_token_secret = ""
 
-    def on_error(self, status):
-        print(status)
+# Basic OAuth handling
+auth = OAuthHandler(consumer_key, consumer_secret)
+auth.set_access_token(access_token, access_token_secret)
+api = API(auth)
 
 if __name__ == '__main__':
-    l = StdOutListener()
-    auth = OAuthHandler(consumer_key, consumer_secret)
-    auth.set_access_token(access_token, access_token_secret)
+    # Makes a MyStreamListener instance
+    myStreamListener = MyStreamListener()
+    myStream = Stream(auth = api.auth, listener = myStreamListener)
```

Comment:
If you're aiming to be PEP 8-compliant, there should not be spaces around the `=`s for keyword parameters. From the [Other Recommendations](https://www.python.org/dev/peps/pep-0008/#other-recommendations) section:

> Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value.